js/components/thumbnailContainer.js

```diff
@@ -0,0 +1,303 @@
+/**
+ * Thumbnail component
+ *
+ * @module Thumbnail
+ */
+var React = require('react'),
+  Utils = require('./utils'),
+  CONSTANTS = require('../constants/constants'),
+  Thumbnail = require('./thumbnail'),
+  ThumbnailCarousel = require('./thumbnailCarousel');
+
+var ThumbnailContainer = React.createClass({
+  getInitialState: function() {
```

Comment:
Can we add spaces between all of these functions?

Reply:
\+